docs/conversion/sakai_20_mysql_conversion.sql

```diff
@@ -197,3 +197,8 @@ UPDATE IGNORE rbc_evaluation re
 
 DROP TABLE messages_with_rubric;
 -- END SAK-43441
+
+-- SAK-42371
+ALTER TABLE rbc_evaluation ADD status INT NOT NULL;
```

Comment:
Is the intention here that this column can only hold single digit ints? Perhaps `TINYINT` is more appropriate.

Reply:
Yes, that is the intention. However, unless I hardcode the JPA annotation to a type, like TINYINT, you can't persuade Hibernate to auto ddl that type. You always get an int. I'd rather just stick with what Hibernate produces, tbh, rather than risking breaking Hibernate/JPA's portability. I can make it tinyint in the conversion, and that works with the code, but auto ddl will always generate an int column.